packages/color-modes/package.json

```diff
@@ -17,12 +17,14 @@
     "@emotion/react": "^11.1.1",
     "@theme-ui/core": "^0.6.1",
     "@theme-ui/css": "0.6.0",
-    "deepmerge": "^4.2.2"
+    "deepmerge": "^4.2.2",
+    "lodash.omit": "^4.5.0"
```

Comment:
Kind of nitpick, but you're only using `omit` to remove `modes`.
It's 2.5kB minified gzipped, and we're already looking pretty big on bundlephobia.

What do you think about changing it to

```ts
const omitModes = (colors: ColorModesScale) => {
  const res = { ...colors };
  delete res.modes;
  return res;
}
```